clayui.com/gatsby-browser.js

```diff
@@ -3,8 +3,28 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
-require('./src/styles/main.scss');
-
 // A stub function is needed because gatsby won't load this file otherwise
 // (https://github.com/gatsbyjs/gatsby/issues/6759)
-exports.onClientEntry = () => {};
+export const onClientEntry = () => {
+	if (process.env.GATSBY_CLAY_NIGHTLY === 'true') {
```

Comment:
Had to pull this out on its own in order for gatsby + webpack to handle these requires correctly.

Reply:
Maybe just adding one more conditional `isNullOrTrue` can solve it? 🤔

```javascript
const isNullOrTrue = (val) => val === 'true' || val === null || val === undefined;
// or
const isNullOrTrue = (val) => val === 'true' || val == null;
```

I don't know if this is related.

Reply:
@matuzalemsteles I think its related to the pre-processing of find/replace environment variables. I remember dealing with this pack on AC with webpack.

Reply:
Hmm, no problem. I think maintaining that will not be a problem. It might even help because it could happen that you saved that key in localStorage and when you access the site it will look different.